packages/forma-36-website/src/components/Docsearch.jsx

```diff
@@ -0,0 +1,55 @@
+import React, { useEffect } from 'react';
+import { css } from '@emotion/core';
+import tokens from '@contentful/forma-36-tokens';
+import { TextInput, Icon } from '@contentful/forma-36-react-components';
+
+const styles = {
+  container: css`
+    position: relative;
+    display: flex;
+    align-items: center;
+    width: 100%;
+    margin-bottom: 0.5rem;
+
+    & .algolia-autocomplete {
+      width: 100%;
+    }
+  `,
+  icon: css`
+    position: absolute;
+    right: 0px;
```

Comment:
Nitpicking so feel free to ignore

```suggestion
    right: 0;
```